spd/losses.py

```diff
@@ -126,10 +128,9 @@ def calc_importance_minimality_loss(
 
 
 def calc_masked_recon_layerwise_loss(
-    model: ComponentModel,
+    model: ComponentModel[Any],
```

Comment:
same qn about Any vs nn.Module.

Reply:
I realise now that this is everywhere, so do a ctrl-f if changing.

Reply:
Oh, Any does map to nn.Module because the class definition is `ComponentModel[T: nn.Module]`? That's pretty gross if so.